cmd/agent/main.go

```diff
@@ -75,6 +76,64 @@ func main() {
 		"working directory to use when spawned via Persistent Worker")
 	flag.Parse()
 
+	// Initialize Sentry
+	var release string
+
+	if version != "unknown" {
+		release = version
+	}
+
+	err := sentry.Init(sentry.ClientOptions{
+		Release:          release,
```

Comment:
```suggestion
		Release:          fmt.Sprintf("cirrus-ci-agent@%s", release),
```

Reply:
Good catch, see https://github.com/cirruslabs/cirrus-ci-agent/pull/264/commits/44507df07eb46c6a7054a2f9af60cd20358cc38e for a fix.